docs/api-proposal/existing-api-routes.md

```diff
@@ -0,0 +1,49 @@
+# React app
+
+GET /
```

Comment:
When these are displayed in markdown, they aren't split over lines (see below)
![image](https://user-images.githubusercontent.com/2686765/76688308-fafb6000-6601-11ea-9573-aa1f387b4432.png) Maybe drop them into a bulleted list or something?

Reply:
Yes, please.